src/frontend/src/flows/pin/pinInfo.ts

```diff
@@ -0,0 +1,86 @@
+import { mainWindow } from "$src/components/mainWindow";
+import { pinStepper } from "$src/flows/pin/stepper";
+import { I18n } from "$src/i18n";
+import { mount, renderPage } from "$src/utils/lit-html";
+import { TemplateResult, html } from "lit-html";
+
+import copyJson from "./pinInfo.json";
+
+const pinInfoTemplate = ({
+  i18n,
+  onContinue,
+  cancel,
+  scrollToTop = false,
+}: {
+  onContinue: () => void;
+  i18n: I18n;
+  cancel: () => void;
+  /* put the page into view */
+  scrollToTop?: boolean;
+}): TemplateResult => {
+  const copy = i18n.i18n(copyJson);
+
+  const slot = html`
+    ${pinStepper({ current: "set" })}
```

Comment:
not sure what's this but just wanted to share that a value called "set" instantly unlock my "this is maybe a reserved keyword pay attention" trigger.

Reply:
This is to set the state of the pin registration steps, which are:
* "Set Pin"
* "Confirm Pin"
* "Captcha"
* "Get Internet Identity"

Since it's currently 4 instead of the intended 3 steps anyway, I'll update that when I need to change the number of steps. 👍

Reply:
Thx